src/main/java/com/github/thedeathlycow/moregeodes/features/ModFeatures.java

```diff
@@ -52,6 +52,14 @@ public static void placeFeaturesInBiomes() {
                 MoreGeodes.CONFIG.generateGypsumPatches(),
                 GenerationStep.Feature.VEGETAL_DECORATION
         );
+
+        if (FabricLoaderImpl.INSTANCE.isModLoaded("ae2")) {
```

Comment:
Use `FabricLoader.getInstance()` instead of using implementation classes, as that may break Quilt compatibility

Reply:
Didn't know Quilt would be sensible to that. Yet again, IDEA proposed `INSTANCE` and I only pressed on `Tab` cuz as all self-respecting dev, I am lazy

Reply:
Quilt has different implementation classes but the API is still the same. It's likely that the class `FabricLoaderImpl` doesn't exist on Quilt.